src/repository/mysql_connection.py

```diff
@@ -0,0 +1,62 @@
+import os
+import aiomysql
+import mysql.connector as mysql_connector
+import mysql.connector
+
+
+HOST = os.environ.get("DB_HOST")
+PORT = int(os.environ.get("DB_PORT"))
+USER = os.environ.get("DB_USER")
+PASSWORD = os.environ.get("DB_PASSWORD")
+DATABASE = os.environ.get("DB_NAME")
+SCRIPT = os.environ.get("DB_SCRIPT")
+    
+def connect(host, port, user, password, database):
+    try:
+        connection = mysql_connector.connect(
+            host=host,
+            port=port,
+            user=user,
+            password=password,
+            database=database,
+            autocommit=True
+        )
+    
+        return connection
+    
+    except mysql.connector.Error as error:
+        print("Error al conectar a la base de datos MySQL:", error)
+        return None
```

Comment:
To start with it is ok... but I think that if we don't have a database connection we would have to crash the application, because it won't work... you can do it here directly... or capture the None that you return in the function that calls it, and apply the corresponding logic.

Reply:
Noted, I have replaced "return None" with good exception handling.